editor-service/src/main/webapp/WEB-INF/views/include/scripts.jsp

```diff
@@ -12,10 +12,9 @@
     <script src="<c:url value='/resources/js/thrift.js' />"></script>
     <script src="<c:url value='/resources/js/transform-to-element-fix.js' />"></script>
 
-    <script src="<c:url value='/resources/js/compiled/utils.js' />"></script>
-    <script src="<c:url value='/resources/js/compiled/root.js' />"></script>
-    <script src="<c:url value='/resources/js/compiled/diagram-core.js' />"></script>
-    <script src="<c:url value='/resources/js/compiled/two-d-model-core.js' />"></script>
+    <script src="<c:url value='/resources/js/compiled/editorCore.js' />"></script>
+    <script src="<c:url value='/resources/js/compiled/twoDModelCore.js' />"></script>
     <script src="<c:url value='/resources/js/compiled/editor.js' />"></script>
+    <script src="<c:url value='/resources/js/compiled/undoRedoPlugin.js' />"></script>
     <script src="<c:url value='/resources/js/compiled/interpreter.js' />"></script>
     <script src="<c:url value='/resources/js/compiled/two-d-model-robots.js' />"></script>
```

Comment:
Newline... You can modify and use script from here [StackOverflow question](http://stackoverflow.com/questions/5922696/add-missing-newlines-in-multiple-files) and add newlines where it is necessary automatically.